javascript/funWithFunctions.md

```diff
@@ -0,0 +1,24 @@
+## Functional Expressions in D&D ##
```

Comment:
TIL feels like it's young and learning who it is but so far, it feels pretty fast and loose and informal and fun. I have no problem with it being story-like.